### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by eddiejaoude** [AccessibleForAll/Support#18]
### Website URL

http://linkfree.eddiehub.io

### GitHub Repo URL

http://github.com/EddieHubCommunity/LinkFree

**Comment by emmalearnscode** [comment]
Thanks Eddie for requesting a review. The video will go live next Saturday. The site is already pretty accessible but here's the things I found which I will add as issues to the repo:

- The menubar and dropdown components are not fully accessible but they're prime react components. Prime react have a statement saying they're working on it so we should keep an eye on when they say the issue is fixed in case we need to use a n updated component.
- The menubar should be moved into a header component on all pages
- Links on the homepage 'Eddie Jaoude' and 'instructions' need distinguishing from normal text. At the moment people who don't perceive color can't tell they are links
- There are several 'learn more' links on the profile page. These should be given some extra info so that screen reader users know what they will learn more about if they click on them.
- Search icon should ideally have a label
